README.md

```diff
@@ -1,78 +1,38 @@
-*   Eventually, the [DevTools core team][] will decide whether the RFC is a candidate for inclusion in DevTools.
-*   RFCs that are candidates for inclusion in DevTools will enter a "final comment period" lasting 7 days. The beginning of this period will be signaled with a
-comment and tag on the RFC's pull request. Furthermore, an email will be sent out the to [DevTools Developer List][dev-developer-tools] to indicate to the broader community that this has entered the final phase.
-*   An RFC can be modified based upon feedback from the [DevTools core team][] and community. Significant modifications may trigger a new final comment period.
-*   An RFC may be rejected by the [DevTools core team][] after public discussion has settled and comments have been made summarizing the rationale for rejection. A member of the [DevTools core team][] should then close the RFC's associated pull request.
-*   An RFC may be accepted at the close of its final comment period. A [DevTools core team][] member will merge the RFC's associated pull request, at which point the RFC will become 'active'.
-*   During a merge an RFC will become numbered and the number will be that of the PR. So PR #5 will have use the `0005-jsx.md` filename eventually.
-
-## The RFC life-cycle
-
-Once an RFC becomes active then authors may implement it and submit the feature as a pull request to the DevTools repos. Becoming 'active' is not a rubber stamp, and in particular still does not mean the change will ultimately be merged; it does mean that the core team has agreed to it in principle and are amenable to the direction.
-
-Furthermore, the fact that a given RFC has been accepted and is 'active' implies nothing about what priority is assigned to its implementation, nor whether anybody is currently working on it.
-
-Modifications to active RFCs can be done in followup PRs. We strive to write each RFC in a manner that it will reflect the final design of the change; but the nature of the process means that we cannot expect every merged RFC to actually reflect what the end result will be at the time of the next major release; therefore we try to keep each RFC
-document somewhat in sync with the change as planned, tracking such changes via followup pull requests to the document.
-
-## Implementing an RFC
+## Reviewing RFCs
 
-The author of an RFC is not obligated to implement it. Of course, the RFC author (like any other developer) is welcome to post an implementation for review after the RFC has been accepted.
+Each week the [DevTools core team][] will attempt to review some set of open RFC issues.
 
-If you are interested in working on the implementation for an 'active' RFC, but cannot determine if someone else is already working on it, feel free to ask (e.g. by leaving a comment on the associated issue).
+We try to make sure that any RFC that we accept is accepted at the Tuesday team meeting, and reported in [DevTools weekly meeting notes][devtools-weekly-meetings].
```

Comment:
I see lots of 'accept' here, what does it mean to accept? Does it mean the core team like the request and 'accept' it and then the whole team has to accept it again?

Reply:
I took this from the previous MD so I'm not sure what the initial intent was. I think we can just collapse this with the previous sentence and have a "Each week the [DevTools core team][] will attempt to review some set of open RFC issues. Reviewed RFCs will also be mentioned in the weekly DevTools team meeting." ?